pr.md

```diff
@@ -5,42 +5,18 @@ This page is really WIP
 
 ## Making a pull request to someone else's repo
 
-Open repo on github
-
-Fork with button in github top corner
-
-Clone YOUR repo not the original
-
+First navigate into the other person's repo. This is ideally done by clicking on his/her profile picture or name. This should show all the repos).
+On top right corner of the screen is a Fork button, which copies the given repo to your account.
```

Comment:
I think you should start by explaining what a fork is, but this is the bare minimum:

```suggestion
On top right corner of the screen is a Fork button, which basically copies the given repo to your account.
```